utils.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import os
+
+from vectara_agentic.agent import Agent
+
+from query_vectara import VectaraQuery
+
+
+def query_vectara(query, convo_id, vectara_prompt, bot_type):
+    if query.startswith('!agentic'):
```

Comment:
I don't think this is a good idea - to have the "!agentic" prefix being required.
Can we please change it so that the user of RAGTime can define (via configuration) whether we use the agentic version or the normal RAG version?